docs/unified-test-documentation/test-matrix.md

```diff
@@ -0,0 +1,148 @@
+# Test matrix
+
+## About
+
+<!--
+Matryca testowa służy do określenia które z zetsawów i przypadków testych
+opisanych w dokumentacji testowej dedykowane są dla danej platformy
+-->
+
+## Test matrix - ASUS KGPE-D16
+
+### Module: dasharo-compatibility
+
+| No.  | Supported test suite                              | Supported test cases                 |
+|:----:|:-------------------------------------------------:|:------------------------------------:|
+| 1.   | coreboot base port                                | All                                  |
```

Comment:
I believe each test group should have ID, an most probably already has, so why not be specific here?

Also `coreboot base port` should be link to test suite documentation.

I assume `coreboot base port` match retirements of our product `Dasharo: coreboot base port`, correct ?

Reply:
1. OK, I will add the ID.
2. OK, I will add a link to the test suite documentation. 
3.  Yes, It matches to Dasharo: coreboot base port